contrib/functions/go/annotate-apply-time-mutations/go.mod

```diff
@@ -2,30 +2,61 @@ module github.com/GoogleContainerTools/kpt-functions-catalog/contrib/functions/g
 
 go 1.17
 
-require sigs.k8s.io/kustomize/kyaml v0.11.1
```

Comment:
Have you run `go mod tidy`?
In go 1.17 and later, direct and transitive dependencies are in 2 `require` stanzas.
It seems you have mixed them together.

Reply:
Ah. I was confused by that change. It wasnt' clear why there's two blocks.